Application/MicroBooter/MicroBooter.cpp

```diff
@@ -3,7 +3,7 @@
 //  <No description>
 //
 //  Microsoft dotNetMF Project
-//  Copyright �2004 Microsoft Corporation
+//  Copyright ©2004 Microsoft Corporation

```

Comment:
@smaillet-ms I've replaced this with the correct copyright symbol. Saved the file with UTF-8 encoding and GitHub still shows a difference at this line. 
If you really think this deserves the effort let me the encoding for the original code page and I'll try to revert this particular change.